deploy/packages/bbp-packages.yaml

```diff
@@ -131,7 +131,7 @@ packages:
       - py-bluepy@0.14.15
       - py-bluepymm@0.7.65 ^neuron%intel@19.0.4
       - py-bluepyopt@1.9.27 ^neuron%intel@19.0.4
-      - py-currentscape@0.0.6
+      - py-currentscape@0.0.9
```

Comment:
This is obsolete. Is there an entry for `py-currentscape` in `deploy/environments/applications.yaml`? If that does not have a version number, it should automatically deploy the "latest" stable version (0.0.9 in this case)

Reply:
There is an entry in deploy/environments/applications.yaml. Should I remove the version number here?

Reply:
No need for that. I'll remove these files in a future PR, they are now not used any longer

Reply:
Alright, then. Thank you.